curvefs/src/client/rpcclient/metaserver_client.cpp

```diff
@@ -532,6 +532,59 @@ bool MetaServerClientImpl::SplitRequestInodes(
     return true;
 }
 
+class BatchGetInodeAttrRpcDone : public MetaServerClientRpcDoneBase {
+ public:
+    BatchGetInodeAttrRpcDone(TaskExecutorDone *done,
+        MetaServerClientMetric *metaserverClientMetric):
+        MetaServerClientRpcDoneBase(done, metaserverClientMetric) {
+            batchDone_ = dynamic_cast<BatchGetInodeAttrTaskExecutorDone*>(done);
+        }
+    virtual ~BatchGetInodeAttrRpcDone() {}
+    void Run() override;
+    BatchGetInodeAttrResponse response;
+
+ protected:
```

Comment:
why protected?

Reply:
I think here protected or private is ok.